app/config/AppConfig.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
```

Comment:
the same warning is here. I'm guessing it appears in the other files too

Reply:
updated for all